saleor/graphql/checkout/dataloaders.py

```diff
@@ -170,7 +170,7 @@ def batch_load(self, keys):
         checkout_by_user_map = defaultdict(list)
         for checkout in checkouts:
             checkout_by_user_map[checkout.user_id].append(checkout)
-        return [checkout_by_user_map.get(user_id) for user_id in keys]
+        return [checkout_by_user_map.get(user_id, []) for user_id in keys]
```

Comment:
It's `defaultdict(list)` that means `[]` is default value if key do not exists. 
```suggestion
        return [checkout_by_user_map.get(user_id) for user_id in keys]
```

Reply:
Actually, when there is no single checkout, the `[checkout_by_user_map.get(user_id) for user_id in keys]` expression will evaluate to `[None]`. With current implementation it returns `[[]]`. 
Later on, in `create_connection_slice` function, we check `if isinstance(iterable, list)`, so I kept current implementation.

Reply:
Maybe we shouldn't use get on `defatultdict`?
```suggestion
        return [checkout_by_user_map[user_id] for user_id in keys]
```
```python
In [1]: from collections import defaultdict

In [2]: checkout_by_user_map = defaultdict(list)

In [3]: checkout_by_user_map.get("fake_key")

In [4]: checkout_by_user_map["fake_key"]
Out[4]: []
```

Reply:
nice! thanks